Проверка рабочего процесса

SystemAgainst opened the pull request:
Here was added: 

1. Name of tutor
2. Connection between main.js and index.html
3. Comment for footer

---
:mortar_board: [Проверка рабочего процесса](https://up.htmlacademy.ru/javascript/27/user/2148431/tasks/5)

:boom: https://htmlacademy-javascript.github.io/2148431-keksobooking-27/2/

Commit:
1. Add name of tutor
2. Add comment to footer

SystemAgainst committed:
Add name of tutor.

vmolchanov commented on index.html:
```diff
@@ -5,6 +5,7 @@
   <link rel="stylesheet" href="css/normalize.css">
   <link rel="stylesheet" href="css/style.css">
   <link rel="shortcut icon" href="favicon.ico" type="image/x-icon">
+  <script type="module" defer src="js/main.js"></script>
```
Этого в задании не нужно было делать

vmolchanov commented on package.json:
```diff
@@ -1,6 +1,7 @@
 {
   "name": "keksobooking",
   "version": "27.0.0",
+  "type": "module",
```
Этого в задании не нужно было делать

vmolchanov commented on Readme.md:
```diff
@@ -1,7 +1,7 @@
 
 * Студент: [Тигран Джамалян](https://up.htmlacademy.ru/javascript/27/user/2148431).
-* Наставник: `Неизвестно`.
+* Наставник: `Владислав Молчанов`.
```
Имя должно быть ссылкой в формате markdown (аналогично тому, как написано выше для тебя)

SystemAgainst committed:
In this commit was realized:
1. connection with index.html
2. function that get random positive integer number
3. function that get random positive float number

SystemAgainst committed:
Corrected mentor's name and added link to his profile.

SystemAgainst committed:
Corrected mentor's name and added link to the profile.

SystemAgainst committed:
Delete code from main.js because of no need in this task.

SystemAgainst committed:
Update Readme.md

vmolchanov reviewed (approved):
👍
